src/index.js

```diff
@@ -0,0 +1,82 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+import path from 'path';
+import loaderUtils from 'loader-utils';
+
+export default function fileLoader(content) {
+  this.cacheable && this.cacheable(); // eslint-disable-line no-unused-expressions
```

Comment:
This isn't needed with webpack >= v2.0.0, I know the PR 'just' apply `webpack-defaults` for the moment, so could be done later, but it's a simple change so maybe :)